docs/examples/checkpointing.rst

```diff
@@ -6,9 +6,10 @@ Use checkpointing with ``exit_for_resume()``
 
 
 This is an example how to use checkpointing and restarting jobs using
-:func:`~cluster_utils.exit_for_resume()` when training a neural network with PyTorch.
+:func:`~cluster_utils.client.exit_for_resume()` when training a neural network with
```

Comment:
Hm, is it possible to reference the indirect imports (i.e. `cluster_utils.exit_for_resume()`) instead? We probably would like users to import that instead of using the client package.

Reply:
I now figured out that it works, if the same namespace is used in the autodoc-directives.  That is, there was actually no need to touch the documentation, I reverted the whole commit.